public/partials/report_v2/report-edit.html

```diff
@@ -29,14 +29,13 @@
         <span ng-if="time !== undefined">({{time}}ms)</span>
     </button>
     <button  id="viewSQL" class="btn btn-default" ng-show="user.viewSQL && sql" class="pull-right" ng-click="getSQLPanel();"  style="padding: 3px 10px;">View SQL</button>
+    <button  id="logQuery" class="btn btn-default" class="pull-right" ng-click="logReport();"  style="padding: 3px 10px;">Log Query</button>
 
+    <el ng-show="selectedReport.query.recordLimit == undefined">
```

Comment:
'el' is not a standard HTML element and does not seem to be a valid AngularJS directive. Can you explain ?